### ISSUE TITLE
Undefined variables in simulation.py

### ISSUE THREAD

**Issue by wenzeslaus** [ncsu-landscape-dynamics/pathways-simulation#16]
There are two undefined variables in simulation.py or, possibly, there is some duplication - I was not able to figure out which one it is.

The apparent error is visible in (now separate) [Flake8 check](https://github.com/ncsu-landscape-dynamics/pathways-simulation/runs/602887719):

```
./pathways/simulation.py:152:13: F821 undefined name 'total_stems_inspected_completion'
./pathways/simulation.py:153:13: F821 undefined name 'total_stems_inspected_detection'
```

and also in [tests](https://github.com/ncsu-landscape-dynamics/pathways-simulation/runs/602887940):

```
Run python -m pathways --num-simulations 5 --num-shipments 50 --config-file config.yml
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/runner/work/pathways-simulation/pathways-simulation/pathways/__main__.py", line 25, in <module>
    main()
  File "/home/runner/work/pathways-simulation/pathways-simulation/pathways/app.py", line 135, in main
    pretty=args.pretty,
  File "/home/runner/work/pathways-simulation/pathways-simulation/pathways/simulation.py", line 281, in run_simulation
    pretty=pretty,
  File "/home/runner/work/pathways-simulation/pathways-simulation/pathways/simulation.py", line 152, in simulation
    total_stems_inspected_completion += stems_inspected_completion
UnboundLocalError: local variable 'total_stems_inspected_completion' referenced before assignment
```

This is one of the issues from 9d565f0 I was not able to resolve in ab23353 (#14).

**Comment by kellynm** [comment]
I found the problem - its a typo in the variable names. Can I just push my changes to the master? Or is there a way to have you review it without making a branch? Or should I make a branch?

**Comment by wenzeslaus** [comment]
I now make a branch and PR for (almost) everything. It is more work, but most of the time worth it. Review is not possible without a PR or at least branch.

**Comment by wenzeslaus** [comment]
Fixed in 2f1c037f4 (#20).
